### ISSUE TITLE
wget functionality via external process : error detection needed

### ISSUE THREAD

**Issue by sheridancbio** [cBioPortal/pdb-annotation#29]
PdbScriptsPipelineRunCommand.java has code for downloadfile(), but the error detection of failed downloads is not adequate. The return code / exit code of the wget command must be checked. Also the output streams from the process must be handled.

(These problems also occur for other ProcessBuilder use cases)

**Comment by sheridancbio** [comment]
Just to verify ... do we use wget for the download of ensembl and pdb sequences? Anything else?

**Comment by juexinwang** [comment]
Yes, only for these two. nothing else. In Update, I used basic ftp for wget does not work in that ftp server.

**Comment by juexinwang** [comment]
Done
